app/templates/_confirm_order.hbs

```diff
@@ -2,7 +2,7 @@
   <div class = "organisation-detail small-12 columns">
     <div class="detail">
       <div class="row">
-        <div class="name small-4 columns">Transport method:</div>
+        <div class="name small-4 columns"> {{t "order.transport_details.transport_method"}}:</div>
```

Comment:
I would include the ":" in the translation - this may be different in another language.

Reply:
Sure @steveyken , Thanks 👍